output.js

```diff
@@ -62,6 +63,83 @@ function status(config, state) {
     }
 }
 
+/**
+ * Convert a time to a human readable string
+ * @param {*} config
+ * @param {number} duration Time in ms
+ */
+function formatDuration(config, duration) {
+    let seconds = Math.floor((duration / 1000) % 60);
+    let minutes = Math.floor((duration / (1000 * 60)) % 60);
+    let hours = Math.floor((duration / (1000 * 60 * 60)) % 24);
```

Comment:
This cuts off days, without us showing those. Just removing `% 24` should fix that.